examples/docker/census-on-omnisci/census-omnisci.py

```diff
@@ -25,25 +26,105 @@
 import numpy as np
 
 
-def read():
+def read(filename):
     columns_names = [
-        "YEAR0", "DATANUM", "SERIAL", "CBSERIAL", "HHWT", "CPI99", "GQ", "QGQ", "PERNUM", "PERWT", "SEX",
-        "AGE", "EDUC", "EDUCD", "INCTOT", "SEX_HEAD", "SEX_MOM", "SEX_POP", "SEX_SP", "SEX_MOM2", "SEX_POP2",
-        "AGE_HEAD", "AGE_MOM", "AGE_POP", "AGE_SP", "AGE_MOM2", "AGE_POP2", "EDUC_HEAD", "EDUC_MOM", "EDUC_POP",
-        "EDUC_SP", "EDUC_MOM2", "EDUC_POP2", "EDUCD_HEAD", "EDUCD_MOM", "EDUCD_POP", "EDUCD_SP", "EDUCD_MOM2",
-        "EDUCD_POP2", "INCTOT_HEAD", "INCTOT_MOM", "INCTOT_POP", "INCTOT_SP", "INCTOT_MOM2", "INCTOT_POP2",
+        "YEAR0",
+        "DATANUM",
+        "SERIAL",
+        "CBSERIAL",
+        "HHWT",
+        "CPI99",
+        "GQ",
+        "QGQ",
+        "PERNUM",
+        "PERWT",
+        "SEX",
```

Comment:
A more compact approach.
```suggestion
        *("YEAR0", "DATANUM", "SERIAL", "CBSERIAL", "HHWT", "CPI99", "GQ", "QGQ", "PERNUM", "PERWT", "SEX"),
```

Reply:
It is what black formatting did, I didn't make any changes to the code.

Reply:
I know, and yet it is difficult to read code that has a lot of long columns.

Reply:
Black converts your code into this. Do you really think it is better?
```python
        *(
            "YEAR0",
            "DATANUM",
            "SERIAL",
            "CBSERIAL",
            "HHWT",
            "CPI99",
            "GQ",
            "QGQ",
            "PERNUM",
            "PERWT",
            "SEX",
        ),
```

Reply:
In the case tuple should be shorter: `*("YEAR0", "DATANUM", "SERIAL", "CBSERIAL", "HHWT", "CPI99", "GQ", "QGQ")`.

Reply:
In my opinion this cosmetic improvement is questionable and is done at an expense of clarity that is important in an example. I don't agree with this change.